commonimages/base/shared.auto.tfvars

```diff
@@ -0,0 +1,36 @@
+
+ami_name_prefix = "base"
```

Comment:
It used to be `baseimage` now it's `base`. It's fine with me, as long as you're cool with it @wullub